src/store/sagas.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  deleteTodos,
+  DELETE_TODOS_REQUESTED,
+} from '@/modules/todo/redux/todo-slice.sagas.delete'
+import { takeEvery } from 'redux-saga/effects'
+
+function* mySaga() {
+  yield takeEvery(DELETE_TODOS_REQUESTED, deleteTodos)
```

Comment:
As we discusses it's better to have takeEery and such i specific redux saga file